<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QnDeprecatedUserManagementWidget</class>
 <widget class="QWidget" name="QnDeprecatedUserManagementWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>400</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="mainVerticalLayout" stretch="0,0">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>16</number>
       </property>
       <item>
        <widget class="QLineEdit" name="filterLineEdit">
         <property name="placeholderText">
          <string>Search users</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="DropdownButton" name="filterButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string notr="true">&lt;User Filter&gt;</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QStackedWidget" name="searchWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="usersPage">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="nx::vms::client::desktop::TreeView" name="usersTable">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="focusPolicy">
            <enum>Qt::TabFocus</enum>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustToContents</enum>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::NoSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="rootIsDecorated">
            <bool>false</bool>
           </property>
           <property name="uniformRowHeights">
            <bool>true</bool>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <property name="allColumnsShowFocus">
            <bool>true</bool>
           </property>
           <attribute name="headerStretchLastSection">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_1">
           <property name="spacing">
            <number>8</number>
           </property>
           <property name="topMargin">
            <number>8</number>
           </property>
           <property name="bottomMargin">
            <number>8</number>
           </property>
           <item>
            <widget class="QPushButton" name="enableSelectedButton">
             <property name="text">
              <string>Enable</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="disableSelectedButton">
             <property name="text">
              <string>Disable</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deleteSelectedButton">
             <property name="text">
              <string>Delete...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="forceSecureAuthButton">
             <property name="text">
              <string>Force Secure Authentication</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_1">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="nothingFoundPage">
        <layout class="QGridLayout" name="gridLayout" rowstretch="2,0,3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Nothing found</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="createUserButton">
       <property name="text">
        <string>New User...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rolesButton">
       <property name="text">
        <string>Edit Roles...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="nx::vms::client::desktop::HintButton" name="ldapTooltip"/>
     </item>
     <item>
      <widget class="QPushButton" name="ldapSettingsButton">
       <property name="text">
        <string>LDAP Settings...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fetchButton">
       <property name="text">
        <string>Fetch Users from LDAP...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>nx::vms::client::desktop::TreeView</class>
   <extends>QTreeView</extends>
   <header location="global">nx/vms/client/desktop/common/widgets/tree_view.h</header>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::HintButton</class>
   <extends>QPushButton</extends>
   <header location="global">nx/vms/client/desktop/common/widgets/hint_button.h</header>
  </customwidget>
  <customwidget>
   <class>DropdownButton</class>
   <extends>QPushButton</extends>
   <header>nx/vms/client/desktop/common/widgets/dropdown_button.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>filterLineEdit</tabstop>
  <tabstop>filterButton</tabstop>
  <tabstop>usersTable</tabstop>
  <tabstop>enableSelectedButton</tabstop>
  <tabstop>disableSelectedButton</tabstop>
  <tabstop>deleteSelectedButton</tabstop>
  <tabstop>forceSecureAuthButton</tabstop>
  <tabstop>createUserButton</tabstop>
  <tabstop>rolesButton</tabstop>
  <tabstop>ldapSettingsButton</tabstop>
  <tabstop>fetchButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
